package.json

```diff
@@ -17,6 +17,7 @@
     "gulp-babel": "^5.3.0",
     "gulp-cache": "^0.3.0",
     "gulp-concat": "^2.6.0",
+    "gulp-expect-file": "^0.0.7",
```

Comment:
I was trying out [this](http://stackoverflow.com/a/22433775/1435958) solution. Didn't seem to work. I forgot to remove it. I'll remove it from the `package.json` and merge.